index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE>
+<html>
+<head>
+     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Trial project</title>
+<link rel="stylesheet" href="styles.css"
```

Comment:
- [x] You have done a very good job requesting this change, however, as mentioned by the previous reviewer,  Please use kind words, be clear, and always give a clear explanation of why the requested change is necessary.

for example, you could say: :arrow_down: 

- [ ] Good job so far, :clap: , however, Please ensure that you always use proper indentation in code, Indentation in code is a formatting convention used in programming languages to visually structure and organize the code blocks. It involves adding consistent spaces or tabs at the beginning of lines to indicate the hierarchy and relationships between different parts of the code.  Well-indented code is easier to maintain. When making changes to the code, developers can quickly understand which lines of code belong to specific blocks, reducing the risk of introducing errors. You may refer to this [Resouce](https://www.freecodecamp.org/news/how-to-indent-in-html-and-why-it-is-important/) for more understanding. :slightly_smiling_face: